app/queries/walkshed_query.rb

```diff
@@ -50,19 +51,19 @@ def paths_sql(school_id, network='survey_network_walk', miles=1.5)
       SELECT ogc_fid, geometry, route.cost from #{network} as w
       JOIN
       (SELECT * FROM
-         pgr_drivingdistance(
+         pgr_drivingDistance(
               'SELECT ogc_fid as id, source, target, miles AS cost
                FROM #{network}
-               WHERE geometry && ST_Buffer(ST_Envelope(#{school}), 8000)'
-              , #{closest_street}, #{miles}, false, false
+               WHERE geometry && ST_Buffer(ST_Envelope(#{school}), 80000)'
```

Comment:
Oh wait! I changed this line around seeing it helped fix the error. Can we change this back to 8000 first? Sorry.